Sources/SpeziQuestionnaire/SpeziQuestionnaire.docc/Model.md

```
@@ -0,0 +1,42 @@
+#
+# SPDX-License-Identifier: MIT
+#       
+-->
+
+Reusable Questionnaire Definition
+
+## Overview
+
+A questionnaire.
+
+## Overview
```

markdownlint-cli2 (0.22.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)